website/docs/r/lb_listener_rule.html.markdown

```diff
@@ -295,16 +295,14 @@ One or more condition blocks can be set per rule. Most condition types can only
 
 Condition Blocks (for `condition`) support the following:
 
-* `field` - (Optional, **DEPRECATED**) The type of condition. Valid values are `host-header` or `path-pattern`. Must also set `values`.
-* `values` - (Optional, **DEPRECATED**) List of exactly one pattern to match. Required when `field` is set.
```

Comment:
minor q: should these removals be added to the upgrade guide? or if the documentation doesn't reference them any more should it already be covered?

Reply:
Indeed! This was actually covered earlier 👍 https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/version-3-upgrade#resource-aws_lb_listener_rule